### PR TITLE
[counter]: add support to enable/disable counter query

### PR DESCRIPTION
**Pull request by sihuihan88**
Signed-off-by: Sihui Han <sihan@microsoft.com>

**What I did**
Add support to enable/disable counter query
**Why I did it**
Add the flexibility to enable or disable counter query.
**How I verified it**
Tested on DUT
**Details if related**

### CODE REVIEW

**Comment by sihuihan88** [comment]
depends on https://github.com/Azure/sonic-swss-common/pull/202 and https://github.com/Azure/sonic-sairedis/pull/322

**Comment by lguohan** [comment]
retest this please

File: orchagent/flexcounterorch.cpp [hunk 1]
```diff
@@ -61,13 +62,22 @@ void FlexCounterOrch::doTask(Consumer &consumer)
                 const auto &field = fvField(valuePair);
                 const auto &value = fvValue(valuePair);
 
-                if (field ==  POLL_INTERVAL_FIELD)
+                if (field == POLL_INTERVAL_FIELD)
```
Comment (lguohan) [inline orchagent/flexcounterorch.cpp]: I do not recall we need the FIELD suffix now, when did we introduce them?

Comment (sihuihan88) [reply orchagent/flexcounterorch.cpp]: They were introduced before. I can have a separate PR to clean it up later if that is preferred

**Pull request merged**